cmd/sidecar-proxy/main.go

```diff
@@ -208,7 +208,7 @@ func run(log *logrus.Entry) error {
 	if !ok {
 		return errors.New("missing access token")
 	}
-	insecureProxyValue, _ := os.LookupEnv("SKIP_CERTIFICATE_VALIDATION")
+	insecureProxyValue, _ := os.LookupEnv("SKIP_CERTIFICATE_VALIDATION") || os.LookupEnv("INSECURE")
```

Comment:
You'll need to set 2 different variables and then check if either is true

Reply:
@tdawe I guess you are looking at the old commit . I have used two different variable

Reply:
Yes, sorry I was looking at the old one :)